mlir/test/Dialect/Linalg/element_wise/invalid.mlir

```diff
@@ -0,0 +1,54 @@
+// RUN: mlir-opt %s -split-input-file -verify-diagnostics
+func.func @misspelt_op_div(%A : memref<16x8xf32>, %B: memref<16x8xf32>, %C: memref<16x8xf32>) {
+  // expected-error@+3 {{expected ::mlir::linalg::ElementwiseFn to be one of: exp, log, abs, ceil, floor}}
```

Comment:
1. Some errors start with `'linalg.elementwise' op` and some without. Please be consistent.
2. Why is it printing the C++ name (`::mlir::linalg::ElementwiseFn`) rather than some MLIR equivalent?